### ISSUE TITLE
Not working after installation

### ISSUE THREAD

**Issue by sensorario** [phparkitect/arkitect#156]
### Bug Report

Doe not work after installation.

|    Q            |   A
|---------------- | ------
| BC Break        | no
| Library Version | 0.1.13
| PHP version     | 8.0.0

#### Summary

Just installed in my machine using `composer require phparkitect/phparkitect`. Using command `./vendor/bin/phparkitect check` I see the following error message: "PHP Warning:  Undefined array key "version" in /Users/simonegentili/Development/github.com/sensorario/prova-architect/vendor/phparkitect/phparkitect/src/CLI/Version.php on line 24". And also this error message: "PHP Fatal error:  Uncaught TypeError: Arkitect\CLI\Version::get(): Return value must be of type string, null returned in /Users/simonegentili/Development/github.com/sensorario/prova-architect/vendor/phparkitect/phparkitect/src/CLI/Version.php:24".

My composer.json is

```
{
    "require": {
        "phparkitect/phparkitect": "^0.1.13"
    }
}
```
#### Current behavior

It is simple not working.

#### How to reproduce

Just install `phparkitect` and then run './vendor/bin'/phparkitect'

#### Expected behavior

I dont know actually.

**Comment by sensorario** [comment]
Trying to run test I can see that my php version is too recent.

```
1) Arkitect\Tests\Unit\Analyzer\FileParserTest::test_parse_file_with_name_match
PHP < 8.0 is required.
```

**Comment by AlessandroMinoccheri** [comment]
I think that it's not a PHP version problem but a problem when we try to get the version of phparkitect for the CLI.
I-m working on it here #157

**Comment by AlessandroMinoccheri** [comment]
We have released a new version `0.1.14` with the fix.
At the moment I close this issue but feel free @sensorario to write again here if you have the same problem.
